vmdb/app/assets/javascripts/miq_application.js

```diff
@@ -542,7 +547,7 @@ function miqSetButtons(count, button_div) {
               } else {
                 tb.enableItem(button);
               }
-            } else if (onwhen = "1") {
+            } else if (onwhen == "1") {
```

Comment:
@martinpovolny Checked by selecting multiple dialogs. Seems to work correctly for me.